app/src/main/res/layouts/main_layouts/layout-land/filter_bottom_sheet.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Pretty sure this layout can be simplified using Constraint layout but its not part of the project yet. Will look to do as a follow up